spec/http/1.0/openapi.json

```diff
@@ -66,14 +66,14 @@
                       "type": "string",
                       "format": "uri"
                     },
-                    "tools": {
+                    "items": {
```

Comment:
Renamed for expected consistency with a future pagination interface.

Reply:
An example of where this is useful? Does it help somehow with parameterizing types on the client side for static type purposes?